### DISCUSSION TITLE
How to make a tag's target="_blank" in typography.

### DISCUSSION THREAD

**Discussion by OmarAfet** [tailwindlabs/tailwindcss#12329] [Help]
Hello, I'm struggling with typography and am trying to customize it. One of the tasks I'm currently working on is setting the `a` tag's `target` attribute to `"_blank"`. In typography, when creating a markdown post containing links, clicking on them replaces the original website instead of opening the link in a new tab (using `target="_blank"`).

**Comment by wongjn** [comment]
`@tailwind/typography` does not handle translation of markdown to HTML. You'd need to look at adjusting whatever technologies you *are* using to do this to add the `target="_blank"` attribute. Consider seeking assistance elsewhere that is more relevant to your technology stack in use.
